### PR TITLE
[CP] Add citizen action Ical export

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/Serializer/CitizenActionICalHandler.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                $organizer->getFirstName(),
+                mb_strtoupper($organizer->getLastName()),
+                $organizer->getEmailAddress()
+            );
+        }
+
+        $visitor->setRoot($data);
+    }
+
+    private function formatDate(\DateTimeInterface $date): string
```
Comment (HeahDude) [inline src/Serializer/CitizenActionICalHandler.php]: Can be static.

File: tests/Controller/CitizenActionControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    public function testExportIcalAction(): void
+    {
+        $uuid = LoadCitizenActionData::CITIZEN_ACTION_3_UUID;
+        /** @var CitizenAction $citizenAction */
+        $citizenAction = $this->getRepository(CitizenAction::class)->findOneBy(['uuid' => $uuid]);
+
+        $this->client->request(Request::METHOD_GET, '/action-citoyenne/2017-12-30-projet-citoyen-3/ical');
+
+        $this->isSuccessful($response = $this->client->getResponse());
+        $this->assertSame('attachment; filename='.$citizenAction->getFinishAt()->format('Y-m-d').'-projet-citoyen-3.ics', $response->headers->get('Content-Disposition'));
```
Comment (HeahDude) [inline tests/Controller/CitizenActionControllerTest.php]: Please use a sprintf for readability.

File: src/Serializer/CitizenActionICalHandler.php (hunk 1 above)
Comment (B-Galati) [reply src/Serializer/CitizenActionICalHandler.php]: done

File: tests/Controller/CitizenActionControllerTest.php (hunk 2 above)
Comment (B-Galati) [reply tests/Controller/CitizenActionControllerTest.php]: done

**Pull request merged**